cdap-docs/reference-manual/source/http-restful-api-v3/introduction.rst

```diff
@@ -0,0 +1,179 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+    :copyright: Copyright © 2014 Cask Data, Inc.
+
+.. _http-restful-api-introduction:
+
+===========================================================
+Introduction
+===========================================================
+
+.. highlight:: console
+
+.. _http-restful-api-conventions:
+
+Conventions
+============
+
+In this API, *client* refers to an external application that is calling CDAP using the HTTP interface.
+*Application* refers to a user Application that has been deployed into CDAP.
+
+.. rubric:: Base URL
+
+All URLs referenced in this API (with the exception of those in the 
+Namespace API) have this base URL::
+
+  http://<host>:<port>/v3/namespaces/<namespace-id>
```

Comment:
I like this approach. However, we may then want to group APIs into two categories - namespaced and non-namespaced. I'm collecting APIs that are non-namespaced at [CDAP-1180](https://issues.cask.co/browse/CDAP-1180). There are [Preferences and Metrics APIs](https://wiki.continuuity.com/display/ENG/v3+APIs) that do not follow this convention. Maybe we need to address them separately. 

A better approach could be to have two base urls? 
``<base-url> = http://<host>:<port>/v3``
``<namespaced-base-url> = http://<host>:<port>/v3/namespaces/<namespace-id>``?